src/main/java/com/structures/fake/FakeWorld.java

```diff
@@ -1,36 +1,55 @@
 package com.structures.fake;
 
 import net.minecraft.block.state.IBlockState;
-import net.minecraft.init.Blocks;
 import net.minecraft.profiler.Profiler;
 import net.minecraft.tileentity.TileEntity;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.World;
 import net.minecraft.world.WorldProvider;
-import net.minecraft.world.WorldProviderSurface;
-import net.minecraft.world.chunk.Chunk;
 import net.minecraft.world.chunk.IChunkProvider;
 import net.minecraft.world.storage.ISaveHandler;
 import net.minecraft.world.storage.WorldInfo;
+import org.jetbrains.annotations.NotNull;
 
-public class FakeWorld extends World{
+/**
+ * Holds a fake world.
+ */
+public class FakeWorld extends World
+{
 	
 	private IBlockState blockState;
-	
-	public FakeWorld(IBlockState blockState, ISaveHandler saveHandlerIn, WorldInfo info, WorldProvider providerIn, Profiler profilerIn, boolean client){
+
+    /**
+     * Creates a fake world.
+     * @param blockState with the blockState.
+     * @param saveHandlerIn a saveHandler.
+     * @param info additional info.
+     * @param providerIn worldProvider.
+     * @param profilerIn profiler.
+     * @param client and if is client.
+     */
+	public FakeWorld(IBlockState blockState, ISaveHandler saveHandlerIn, WorldInfo info, WorldProvider providerIn, Profiler profilerIn, boolean client)
+    {
 		super(saveHandlerIn, info, providerIn, profilerIn, client);
 		this.blockState = blockState;
 	}
 
-	@Override
-	protected IChunkProvider createChunkProvider() {
-		//Not needed
+	@NotNull
+    @Override
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Annotation 'Override' have incorrect indentation level 4, expected level should be 1. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.annotation.AnnotationLocationCheck)